src/index.mjs

```diff
@@ -132,12 +125,7 @@ const build = (statics) => {
 						case TAG_END:
 							commit();
 							if (mode !== MODE_SKIP) {
-								if (!props) {
-									out += ',null';
-								}
-								else {
-									out += ',' + props + propsClose + spreadClose;
-								}
+								out += ',' + (props + propsClose + spreadClose || null);
```

Comment:
Just cause I’m curious... how are these two equivalent. `props + propsClose + spreadClose` is falsy if `props` is falsy?